src/modules/Lan.rb

```diff
@@ -713,13 +725,13 @@ def Import(settings)
     def Export
       profile = Y2Network::AutoinstProfile::NetworkingSection.new_from_network(yast_config)
       ay = {
-        "dns"                  => profile.dns ? profile.dns.to_hashes : {},
-        "net-udev"             => profile.udev_rules ? profile.udev_rules.udev_rules.map(&:to_hashes) : [],
-        "s390-devices"         => profile.s390_devices ? profile.s390_devices.to_hashes["devices"] : [],
+        "dns"                  => profile&.dns&.to_hashes || {},
```

Comment:
But `profile` cannot be `nil`, we have just constructed it:
https://github.com/yast/yast-network/pull/994/files#diff-f0657dacb6a78e1b8c4dbfc977fc0e54R72-R82

This is the bad thing about RuboCop: it encourages you to make subtle logic changes, hidden among its monstrous near-whitespace diffs.

Reply:
ah, right, I do too much. Lets revert this change.